doc/test-requirements-template.md

```diff
@@ -75,7 +79,7 @@ README may be approved and merged before the TODO item(s) are resolved.
           "scheduler-policy": {
             "config": {
               "name": "limit_group_A_1Gb",
-              "new-leaf": "my_new_value"     # TODO https://github.com/openconfig/public/pull/1234
+              "new-leaf": "my_new_value"
```

Comment:
Remove this line as the canonical OC would not parse with this `new-leaf` as it is not part of the public OC data models.